### ISSUE TITLE
Support Mendeley citations in docx like Zotero citations support (#7840)

### ISSUE THREAD

**Issue by jooyoungseo** [jgm/pandoc#7886]
This is the same request as #7875 and #7840.

[Mendeley](https://www.mendeley.com/download-reference-manager/linux) is another free reference management tool like Zotero, and it's widely used in academia as well.

It would be greatly appreciated if Pandoc could also support this.

I am attaching a sample document: [Mendeley.docx](https://github.com/jgm/pandoc/files/7991126/Mendeley.docx)

**Comment by jgm** [comment]
Here's what their ADDIN looks like:
```
        <W:INSTRTEXT>ADDIN CSL_CITATION
        {"citationItems":[{"id":"ITEM-1","itemData":{"abstract":"Programming
        can be challenging to learn and for visually impaired (VI)
        learners there are numerous additional barriers to the
        learning process. Many modern programming environments are
        inaccessible to VI learners, being difficult or impossible
        to interface with using a screen reader. A review of the
        literature has identified a number of strategies that have
        been employed in the quest to make learning to program
        accessible to VI learners. These can be broadly divided
        into the following categories; auditory and haptic
        feedback, making text-based languages (TBLs) accessible,
        making block-based languages (BBLs) accessible and physical
        artifacts. A common theme among the literature is the
        difficulty VI learners have in gaining an understanding of
        the overall structure of their code. Much of the research
        carried out in this space to date focuses on the evaluation
        of interventions aimed at VI high-school and undergraduate
        students, with limited attention given to the learning
        processes of VI learners. Additionally, the majority of the
        research deals with (TBLs), this is despite the fact that
        most introductory programming courses for primary learners
        use (BBLs). Therefore, further research is urgently needed
        to investigate potential strategies for introducing VI
        children in primary education to programming and the
        learning processes
        involved.","author":[{"dropping-particle":"","family":"Hadwen-Bennett","given":"Alex","non-dropping-particle":"","parse-names":false,"suffix":""},{"dropping-particle":"","family":"Sentance","given":"Sue","non-dropping-particle":"","parse-names":false,"suffix":""},{"dropping-particle":"","family":"Morrison","given":"Cecily","non-dropping-particle":"","parse-names":false,"suffix":""}],"container-title":"International
        Journal of Computer Science Education in
        Schools","id":"ITEM-1","issue":"2","issued":{"date-parts":[["2018","4"]]},"page":"3-13","title":"Making
        Programming Accessible to Learners with Visual Impairments:
        A Literature
        Review","type":"article-journal","volume":"2"},"locator":"123","prefix":"prefix","suffix":"suffix","uris":["http://www.mendeley.com/documents/?uuid=3acf31b7-f77b-4e8c-81b6-2f2c22f05564"]}],"mendeley":{"formattedCitation":"(prefix
        Hadwen-Bennett et al., 2018, p. 123
        suffix)","plainTextFormattedCitation":"(prefix
        Hadwen-Bennett et al., 2018, p. 123
        suffix)","previouslyFormattedCitation":"(prefix
        Hadwen-Bennett et al., 2018, p. 123
        suffix)"},"properties":{"noteIndex":0},"schema":"https://github.com/citation-style-language/schema/raw/master/csl-citation.json"}
```
So, they use CSL much like Zotero.  For the bibliography,
```
ADDIN Mendeley Bibliography CSL_BIBLIOGRAPHY 
```
with no further content.

**Comment by jgm** [comment]
Probably the same code will support both Mendeley and Zotero addins, with minor changes.

**Comment by jgm** [comment]
Closed by e07c0e74ceaadc3974d94801b4c69e8754afc85b
